Sources/SwiftParser/generated/Parser+TokenSpecSet.swift

```diff
@@ -241,11 +318,28 @@ extension AvailabilityLabeledArgumentSyntax {
         return .keyword(.deprecated)
       }
     }
+    
+    @_spi(Diagnostics)
+    public var tokenSyntax: TokenSyntax {
+      switch self {
+      case .message:
+        return .keyword(.message)
+      case .renamed:
+        return .keyword(.renamed)
+      case .introduced:
+        return .keyword(.introduced)
+      case .obsoleted:
+        return .keyword(.obsoleted)
+      case .deprecated:
+        return .keyword(.deprecated)
+      }
+    }
```

Comment:
@ahoppen I was thinking of adding something like this to the tokenSpec. 

It can then be used in the ParseDiagnosticsGenerator like this `LabeledSpecializeEntrySyntax.LabelOptions.allCases.map { $0.tokenSyntax }`

But the thing is, if it's an `identifier` etc. we don't have any value. Should we just pass an empty string or? 

Let me hear what you think